packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGImage/SVGImage.js

```diff
@@ -42,6 +42,10 @@ export default function SVGImage ({
     width: naturalWidth,
   }
 
+  // image does not have the dragMove property and throws a DOM error blowing up rendering
+  if (!move)
+  	delete SubjectImageProps.dragMove
```

Comment:
whitespace.

Reply:
`SubjectImageProps` is undefined here, which crashes the classifier for all projects and workflows.

Reply:
This bug was fixed in some other separate commit so has been removed